docs/oasis-node/genmetricsdocs.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3
```

Comment:
We should move this under `scripts/` instead of having it in docs and name it something like `generate-metrics-docs.py`.